frontend/src/pages/exploreApplication/useIntegratedAppStatus.ts

```diff
@@ -28,8 +29,9 @@ export const useIntegratedAppStatus = (app?: OdhApplication): FetchState<Integra
       isInstalled: false,
       isEnabled: false,
       canInstall: false,
+      variablesValidationStatus: '',
       error: '',
     },
-    { initialPromisePurity: true },
+    { refreshRate: 5000, initialPromisePurity: true },
```

Comment:
```suggestion
    { refreshRate: FAST_POLL_INTERVAL, initialPromisePurity: true },
```
Recommend you just use our environment variable for anything you want "quickly done" -- it's 3 seconds, not 5 -- but it's how we typically fetch for information on a polling system.

`POLL_INTERVAL` is 30s, so if you want a slower poll interval, this is also available.

cc @emilys314

Reply:
@andrewballantyne Thanks, will use `FAST_POLL_INTERVAL`

Reply:
@andrewballantyne Not sure if it is my network issue, 3s is too fast for me. The situation is like, the first API call is triggered but it takes more than 3s to get a response, and now, the second API call is triggered. In this case, there is infinite loading.

Reply:
Yeah I believe you're correct Yu, the poller can start making new requests even when the previous hasn't finished. 

Is there a reason we need to do it that fast?

Reply:
@emilys314 We use the poller here to check the enablement status of the application. If the application is enabled, we hide the `Enable` button on the UI, so it needs to be fast.

Reply:
The interval is removed.